src/api/app/models/user.rb

```diff
@@ -552,9 +552,9 @@ def has_local_permission?(perm_string, object)
     when Project
       logger.debug "running local permission check: user #{login}, project #{object.name}, permission '#{perm_string}'"
 
-      # Users have permissions to manage their own home project
+      # Users have permissions to manage their own home project and it's subprojects
       # This is needed since users sometimes remove themselves from the maintainers of their own home project
-      return true if object.name == home_project_name
+      return true if object.name == "home:#{login}" || object.name.include?("home:#{login}:")
```

Comment:
The `include?` also gives access to `home:someoneelse:branches:home:LOGIN:foo` - I think, `start_with?` would be better here.

Reply:
Ahh good catch! Fixing...